packages/create-sitecore-jss/src/initializers/react-native/index.ts

```diff
@@ -15,6 +15,8 @@ export default class ReactNativeInitializer implements Initializer {
       ? {
           appName: 'sitecore-jss-react-native',
           hostName: 'sitecore-jss-react-native.dev.local',
+          appPrefix: false,
```

Comment:
Remove `appPrefix` - react-native app doesn't support it

Reply:
Fixed